src/sentry/tasks/commit_context.py

```diff
@@ -83,6 +83,21 @@ def queue_comment_task_if_needed(
             )
         return
 
+    if response[0]["state"] == "open":
+        metrics.incr(
+            "github_pr_comment.queue_comment_check.open_pr",
+            sample_rate=1.0,
+        )
+        logger.info(
```

Comment:
we're adding a log too, just wanna be sure as the PR says only metric. will want to watch out if this is super noisy

Reply:
i'll remove the log